elastic/kibana
# [Exploratory View] Adjust labels for percentile calculation and document loaded metric

## Summary

Fixes: https://github.com/elastic/kibana/issues/116685
Fixes: https://github.com/elastic/kibana/issues/116682

This PR resolves missing labels for percentile metric for the Synthetics data type. 
<img width="246" alt="Screen Shot 2021-10-28 at 2 39 30 PM" src="https://user-images.githubusercontent.com/11356435/139499493-bb41b659-4cb0-484e-97a3-73d6b3541933.png">
This PR also changes the label for the Synthetics document load metric from `Document complete (onload)`. To `Page load time`.

### Testing
1. Navigate to the exploratory view, ensuring that you have Synthetics data. 
2. Select the KPI over time report type and Synthetics data type.
3. When selecting metric type, all of the following metrics should be tested:
<img width="304" alt="Screen Shot 2021-10-28 at 2 39 37 PM" src="https://user-images.githubusercontent.com/11356435/139499462-e199499a-3102-4b78-b891-96a504bad4fb.png">
4. Select an individual metric, then in the expanded row select the percentile metric.
5. In the chart legend, ensure all the labels appear appropriately. 
6. When viewing the `Page load time` metric, ensure that the metric label in the series editor is `Page load time`
